<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFishDialog</class>
 <widget class="QDialog" name="EditFishDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Fish</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelName">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name of fish&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>editName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labelCFactor">
       <property name="toolTip">
        <string comment="Weight = c * x ^ 3 / 100" extracomment="Weight = c * x ^ 3 / 100"/>
       </property>
       <property name="whatsThis">
        <string comment="Weight = Weight = c * x ^ 3 / 100" extracomment="Weight = Weight = c * x ^ 3 / 100"/>
       </property>
       <property name="text">
        <string>&amp;Corpulence factor:</string>
       </property>
       <property name="buddy">
        <cstring>editCFactor</cstring>
       </property>
       <property name="toolTipDuration" stdset="0">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="labelPath">
       <property name="text">
        <string>&amp;Path:</string>
       </property>
       <property name="buddy">
        <cstring>editPath</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="editCFactor"/>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QLineEdit" name="editName">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name of fish&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="editPath"/>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="selectButton">
       <property name="text">
        <string>&amp;Select</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="labelShortcut">
       <property name="text">
        <string>Shortcut:</string>
       </property>
       <property name="buddy">
        <cstring>editShortcut</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="editShortcut"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialogButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editCFactor</tabstop>
  <tabstop>editShortcut</tabstop>
  <tabstop>editPath</tabstop>
  <tabstop>selectButton</tabstop>
  <tabstop>dialogButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditFishDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
